lib/membrane/rtp/outbound_rtx_controller.ex

```diff
@@ -20,23 +19,15 @@ defmodule Membrane.RTP.OutboundRtxController do
   @min_rtx_interval 10
 
   @impl true
-  def handle_init(_opts), do: {:ok, %{store: BufferStore.new(), last_rtx_times: %{}}}
+  def handle_init(_opts), do: {:ok, %{store: %{}, last_rtx_times: %{}}}
 
   # Ignores padding packets
   # TODO: Should it?
   @impl true
   def handle_process(:input, buffer, _ctx, state) when byte_size(buffer.payload) > 0 do
-    state
-    |> Map.update!(:store, fn store ->
-      case BufferStore.insert_buffer(store, buffer) do
-        {:ok, new_store} ->
-          maintain_store_size(new_store)
-
-        {:error, :late_packet} ->
-          store
-      end
-    end)
-    |> then(&{{:ok, forward: buffer}, &1})
+    idx = rem(buffer.metadata.rtp.sequence_number, @max_store_size)
+    state = put_in(state, [:store, idx], {System.monotonic_time(:millisecond), buffer})
```

Comment:
One more thing: this changes the behaviour of the controller as we won't retransmit any packet within 10ms. I assume the receiver is using some JitterBuffer and will wait a bit before sending NACK to compensate for possible reorder, thus we should handle first NACK right away.

I even wonder if we should have any `min_rtx_interval` at all, as it seems large retransmissions are only a symptom, not a root cause of our problems